packages/cerebro/examples/simple-logger-cli.js

```
@@ -0,0 +1,81 @@
+            success: (msg) => console.log(`[✨ SUCCESS] ${msg}`),
+            alert: (msg) => console.log(`[🚨 ALERT] ${msg}`),
+            critical: (msg) => console.log(`[💥 CRITICAL] ${msg}`),
+            emergency: (msg) => console.log(`[🆘 EMERGENCY] ${msg}`),
+            trace: console.trace,
+            warning: (msg) => console.log(`[⚠️  WARNING] ${msg}`),
+            raw: console.log,
+            disable: () => {},
+            enable: () => {},
+        };
+
+        cli = new Cerebro("my-cli", { logger: customLogger });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)